src/pages/tutorials/introduction-to-layer-types.md

```diff
@@ -8,4 +8,89 @@ layout: tutorials.hbs
 
 {{ page.data.description }}
 
-Coming Soon.
\ No newline at end of file
+## Feature Layers
+A [Feature Layer](http://esri.github.io/esri-leaflet/api-reference/layers/feature-layer.html) involves pulling down the feature data for a particular layer in the JSON format from an ArcGIS for Server instance to your web app. This means pulling down the information related to an individual feature such as its vertices as well as the attributes. Esri-Leaflet then takes the JSON data and renders it on your map.
```

Comment:
a little bird told me that we're going to go back to just calling it 'ArcGIS ~~for~~ Server'